blog.html

```diff
@@ -9,8 +9,19 @@ <h1 class="title page-title">The Teach LA Blog</h1>
   We love what we do, and we want to share it with you! Here are some insights from our members on how our club operates: how we teach, write code, and create a community!
 </p>
 <p>
-  Interested in more formal reflections on the inner workings of our club? Check out <a class="tla-link" href="{{site.baseurl}}/reports">reports section</a>. Curious in how we handle teaching an artificial intelligence and machine learning class at a high school, or our joint efforts to provide AI education? Peep the <a class="ai-link" href="https://uclaacmai.github.io/blog/" target="_blank" rel="noopener noreferrer">ACM AI blog!</a>
+  Interested in more formal reflections on the inner workings of our club? Check out 
+  <a class="tla-link" href="{{site.baseurl}}/reports">reports section</a>.
+  Curious in how we handle teaching an artificial intelligence and machine learning class at a high school, or our joint efforts to provide AI education? Peep the 
+  <a class="ai-link" href="https://uclaacmai.github.io/blog/" target="_blank" rel="noopener noreferrer">ACM AI blog</a>!
```

Comment:
Is this just a whitespace change? If so, do you mind reversing it?

Reply:
Oh yeah, I apologize for that! I think I changed it personally to see the code better but forgot to change it back.